Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -142,6 +148,13 @@ QString DMainWindow::ShowFileDialog()
 		.arg(SL("*.gcm *.iso *.ciso *.gcz *.wbfs *.elf *.dol *.dff *.tmd *.wad")));
 }
 
+QString DMainWindow::ShowFolderDialog()
+{
+	return QFileDialog::getExistingDirectory(this, tr("Browse for a directory to add"),
+                                             QDir::homePath(),
+                                             QFileDialog::ShowDirsOnly);
+}
```

Comment:
@darkengine-io  fwiw most editors have a nice visual way to display whitespace: VS is ctrl+r,ctrl+w, notepad++ has a ¶ button, etc.